definitions/variable/tag_electricity_input_types.yaml

```diff
@@ -103,14 +103,80 @@
   - Gas|Fossil|w/o CCS:
       description: fossil natural gas (excluding methane from biomass or hydrogenation)
         with freely vented CO2 emissions
+  - Gas|Fossil|OCGT:
+      description: fossil natural gas (excluding methane from biomass or hydrogenation)
+        in an open-cycle gas turbine (OCGT) power plant 
+  - Gas|Fossil|CCGT:
+      description: fossil natural gas (excluding methane from biomass or hydrogenation)
+        in a combined-cycle gas turbine (CCGT) power plant
+  - Gas|Fossil|Gas Engines:
```

Comment:
I know of "Gas Turbines" - do you mean those?

Reply:
No seems to be different
This name was discussed with Konstantin as it is what is used in Genesys-Mod and it seems to be used (konstantin showed me a publication by siemens explaining the difference between gas turbines and gas engines
I sent you the email